core/examples/slip10.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! SLIP-0010 example applet
+
+use bip39::Mnemonic;
+use clap::Parser;
+use mc_core::{
+    account::Account, prelude::RingCtAddress, slip10::Slip10KeyGenerator, subaddress::Subaddress,
```

Comment:
I'm curious- how do you decide whether to import a struct via the `preulde` module or, in this case, the `account` module?